src/components/LmsConfigurations/index.test.jsx

```diff
@@ -163,15 +183,30 @@ describe('<LmsConfigurations /> ', () => {
     LmsApiService.fetchMoodleConfig.mockRejectedValue(notFoundMoodleResponse);
     LmsApiService.fetchBlackboardConfig.mockRejectedValue(notFoundBlackboardResponse);
     LmsApiService.fetchCanvasConfig.mockRejectedValue(notFoundCanvasResponse);
-    LmsApiService.fetchSuccessFactorsConfig.mockResolvedValue(successFactorsResponse);
+    LmsApiService.fetchSuccessFactorsConfig.mockRejectedValue(notFoundSuccessFactorsResponse);
```

Comment:
Was this just wrong before?

Reply:
Sort of. Ultimately this test should have failed, but because it's not a proper integration test, it passed.
Basically, you _could_ have multiple different integration configs (if configured on the backend) and we would store it in the state. However, what the user would see is the first config in the list (which would have been SuccessFactors here). 
Since we aren't currently wanting users to create multiple integration config forms at this point, this test case should just test the expectation of 1 integration returning true.